object-cache.php

```diff
@@ -491,6 +491,14 @@ function salt_keys( $key_salt ) {
 	}
 
 	function __construct() {
+		$this->stats = array(
+			'get' => 0,
+			'get_multi' => 0,
+			'add' => 0,
+			'set' => 0,
+			'delete' => 0,
+		);
+
```

Comment:
One nitpicky thing would be to align the array values. The plugin appears to align the equals in most cases.